OpenRA.Mods.Cnc/Traits/HunterSeeker.cs

```diff
@@ -0,0 +1,62 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Cnc.Traits
+{
+	sealed class HunterSeekerInfo : TraitInfo<HunterSeeker>
+	{
+	}
```

Comment:
```suggestion
	sealed class HunterSeekerInfo : TraitInfo
	{
		[Desc("Valid target relationships.")]
		public readonly PlayerRelationship TargetRelationships = PlayerRelationship.Enemy;

		public override object Create(ActorInitializer init) { return new HunterSeeker(init.Self, this); }
	}
```

Do add player relationships to make this trait more reusable